rsconnect/main.py

```diff
@@ -821,6 +827,105 @@ def deploy_manifest(name, server, api_key, insecure, cacert, new, app_id, title,
     )
 
 
+@server_args
+@content_args
+@click.option(
+    "--entrypoint",
+    "-e",
+    help=("The module and executable object which serves as the entry point for the deployment."),
+)
+@click.option(
+    "--exclude",
+    "-x",
+    multiple=True,
+    help=(
+        "Specify a glob pattern for ignoring files when building the bundle. Note that your shell may try "
+        "to expand this which will not do what you expect. Generally, it's safest to quote the pattern. "
+        "This option may be repeated."
+    ),
+)
+@click.argument("directory", type=click.Path(exists=True, dir_okay=True, file_okay=False))
+@click.argument(
+    "extra_files",
+    nargs=-1,
+    type=click.Path(exists=True, dir_okay=False, file_okay=True),
+)
+@click.argument("path", type=click.Path(exists=True, dir_okay=True, file_okay=True))
+def deploy_html(
+    name,
+    server,
+    api_key,
+    insecure,
+    cacert,
+    new,
+    app_id,
+    title,
+    verbose,
+    path,
+    env_vars,
+    entrypoint,
+    extra_files,
+    excludes,
+):
+    set_verbosity(verbose)
+
+    entrypoint = entrypoint or infer_entrypoint(path, "text/html")
+    manifest = make_html_manifest(entrypoint)
+
+    with cli_feedback("Checking arguments"):
+        connect_server = _validate_deploy_to_args(name, server, api_key, insecure, cacert)
+        manifest = validate_manifest_file(manifest)
+        app_store = AppStore(manifest)
+
+        (
+            app_id,
+            deployment_name,
+            title,
+            default_title,
+            app_mode,
+            package_manager,
+        ) = gather_basic_deployment_info_from_manifest(connect_server, app_store, manifest, new, app_id, title)
+
+    click.secho('    Deploying %s to server "%s"' % (manifest, connect_server.url))
+
+    if package_manager == "conda":
+        with cli_feedback("Ensuring Conda is supported"):
+            check_server_capabilities(connect_server, [is_conda_supported_on_server])
+
+    with cli_feedback("Creating deployment bundle"):
+        try:
+            bundle = make_html_bundle(path, entrypoint, extra_files, excludes)
+        except IOError as error:
+            msg = "Unable to include the file %s in the bundle: %s" % (
+                error.filename,
+                error.args[1],
+            )
+            if error.args[0] == errno.ENOENT:
+                msg = "\n".join(
+                    [
+                        msg,
+                        "Since the file is missing but referenced in the manifest, "
+                        "you will need to\nregenerate your manifest.  See the help "
+                        'for the "write-manifest" command or,\nfor non-Python '
+                        'content, run the "deploy other-content" command.',
```

Comment:
This case is also not needed, because this is a generated manifest (so we know the files exist) vs. a user-provided manifest which might be out of date.